catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/CompositeEndBrickMoveTest.kt

```diff
@@ -0,0 +1,205 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick.app
+
+import android.view.View
+import android.view.ViewConfiguration
+import androidx.annotation.NonNull
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.GeneralLocation
+import androidx.test.espresso.action.MotionEvents
+import androidx.test.espresso.action.Press
+import org.catrobat.catroid.R
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import org.catrobat.catroid.ProjectManager
+import org.catrobat.catroid.content.Project
+import org.catrobat.catroid.content.Script
+import org.catrobat.catroid.content.Sprite
+import org.catrobat.catroid.content.StartScript
+import org.catrobat.catroid.content.bricks.ChangeSizeByNBrick
+import org.catrobat.catroid.content.bricks.IfThenLogicBeginBrick
+import org.catrobat.catroid.content.bricks.RepeatBrick
+import org.catrobat.catroid.test.utils.TestUtils
+import org.catrobat.catroid.ui.SpriteActivity
+import org.catrobat.catroid.uiespresso.util.rules.FragmentActivityTestRule
+import org.hamcrest.Matcher
+import org.hamcrest.core.IsAnything
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class CompositeEndBrickMoveTest {
+    private lateinit var project: Project
+    private val oneBlockDown: Float = -310f
+    private val oneBlockUp: Float = 360f
+    private lateinit var ifBrick: IfThenLogicBeginBrick
+
+    @Rule
+    @JvmField
+    var baseActivityTestRule = FragmentActivityTestRule(
+        SpriteActivity::class.java,
+        SpriteActivity.EXTRA_FRAGMENT_POSITION,
+        SpriteActivity.FRAGMENT_SCRIPTS
+    )
+
+    @Before
+    @Throws(Exception::class)
+    fun setUp() {
+        createProject(this.javaClass.simpleName)
+        baseActivityTestRule.launchActivity()
+    }
+
+    @Test
+    fun testMoveEndBrick() {
+        Assert.assertEquals(ifBrick.nestedBricks.size, 1)
```

Comment:
Here the expected and the actual value are switched, should be the other way around.